lib/OpenSSL/NativeLib.rakumod

```diff
@@ -28,20 +30,8 @@ sub crypto-lib is export {
 # and use it if it exists, otherwise copy the name mangled file to this location but using the
 # original unmangled name.
 # XXX: This should be removed when CURI/%?RESOURCES gets a mechanism to bypass name mangling
-use nqp;
 sub dll-resource($resource-name) {
-    my $content-id    = nqp::sha1($resource-name);
-    my $dll-directory = $*TMPDIR.add($content-id);
-    my $dll-resource  = $dll-directory.add($resource-name);
-
-    unless $dll-resource.e {
-        mkdir $dll-directory unless $dll-directory.e;
-        my $resource = %?RESOURCES{$resource-name};
-        my $resource-data := %?RESOURCES{$resource-name}.slurp(:bin, :close);
-        $dll-resource.spurt($resource-data, :bin, :close);
-    }
-
-    $dll-resource.absolute
+    load-resource-to-path($resource-name).absolute
```

Comment:
I'm not sure `Resource::Wrangler` is ideal to use here if it indeed needs to generate a unique path part every time it is run since that means on Windows we'd constantly be copying the OpenSSL libraries. In my original code above it would probably have been better to `$*HOME` and the prefix instead of `$*TMPDIR` to avoid security issues, but it has the advantage of putting the file in a deterministic spot so it is ideally only ever copied once.

Reply:
FWIW, on Windows $*TMPDIR is already underneath a user's folder.